flows/codelingo/review/review/cli.go

```diff
@@ -166,16 +169,67 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 		return nil, nil, nil, nil, errors.Trace(err)
 
 	}
+	// Build `path` for go projects as "reponame" for a local repo or "github.com/username/reponame" for a github repo
+	// path variable is initially empty and will remain empty for none go repos
+	path := ""
+
+	// Find the current working directory, i.e. where user executed `lingo run review` command on the repo
+	wd, err := os.Getwd()
```

Comment:
You should use the root of the git repo instead, since the review can be run from a subdirectory.